waspc/src/Wasp/Generator/SdkGenerator/Server/JobGenerator.hs

```diff
@@ -91,15 +91,14 @@ genJob (jobName, job) =
 getJobExecutorImportPath :: JobExecutor -> Path Posix (Rel r) File'
 getJobExecutorImportPath PgBoss = makeSdkImportPath [relfileP|server/jobs/core/pgBoss|]
 
-getImportJsonForJobDefinition :: String -> Aeson.Value
+getImportJsonForJobDefinition :: String -> Generator Aeson.Value
 getImportJsonForJobDefinition jobName =
   GJI.jsImportToImportJson $
     Just $
       JI.JsImport
         { JI._path = JI.ModuleImportPath $ makeSdkImportPath [relfileP|server/jobs|],
           JI._name = JI.JsImportField jobName,
-          -- NOTE: We are using alias to avoid name conflicts with user defined imports.
```

Comment:
Ok, so we are not aliasing our names anymore, because we are aliasing their names.

I wonder how this will affect the error messages? Will they now be seeing stuff like `getTasks__userDefined` in the error messages? If so, it would be better that we mangle our names, and not theirs, when possible.

Also, I remember at some point in the past, I was worrying about the situation where we embed their code directly into the template (so not just importing it, but actually embedding it) -> then there was a lot of opportunity for conflicts with our names in that file. I think we don't have such situations any more though, since we don't embed their code directly -> we had that as a feature in Wasp in very early days and that has been removed.